packages/zipkin-instrumentation-koa/test/koaMiddlewareTest.js

```diff
@@ -0,0 +1,200 @@
+const fetch = require('node-fetch');
+const {Tracer, ExplicitContext} = require('zipkin');
+const Koa = require('koa');
+
+const zipkinMiddleware = require('../src/koaMiddleware');
+
+describe('zipkinMiddlewareTest', () => {
```

Comment:
let's redo this as the recently merged integration tests as it will smoke out more problems and also cause less rewrite work when we move to v2 style api

Reply:
Sure, was going to redo this anyway.